wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,57 +1,83 @@
-| ![](/wiki/shared/judgement/osu!/hit300k.png "300 Katu") | 300 (喝) Katu or Katsu | A variant of the 300 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 300 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 300. |
-| ![](/wiki/shared/judgement/osu!/hit100.png "100") | 100 | The second best achievable result for a hit object. Usually denoted with a green colour. |
-| ![](/wiki/shared/judgement/osu!/hit100k.png "100 Katu") | 100 (喝) Katu or Katsu | A variant of the 100 result. Originates from [Osu! Tatakae! Ouendan!](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan), called Beat! in [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents). Granted upon hitting a 100 on the last element in a combo in which the player has scored at least one 100, but no 50s or misses. Getting a Katu will give a small health bonus on top of the standard health increase for a 100. |
-| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
-| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
+## Judgements
+
+A **judgement** (a.k.a. **hit result**) is the outcome of interacting with a [hit object](/wiki/Gameplay/Hit_object) during its hit window. Score and accuracy are calculated based on which judgements are received.
+
+These are commonly referred to by their score value (except for misses), i.e. a Great is usually called a "300" and so on.
+
+| Image | Name | [Hit value](/wiki/Gameplay/Score/ScoreV1/osu!) | [Accuracy](/wiki/Gameplay/Accuracy#osu!) | Hit window (ms) |
+| :-: | :-: | --: | --: | :-- |
+| ![](/wiki/shared/judgement/osu!/hit300.png) | Great | 300 | 100% | `80 - 6 * OD` |
+| ![](/wiki/shared/judgement/osu!/hit100.png) | Ok | 100 | 33.33% | `140 - 8 * OD` |
+| ![](/wiki/shared/judgement/osu!/hit50.png) | Meh | 50 | 16.67% | `200 - 10 * OD` |
+| ![](/wiki/shared/judgement/osu!/hit0.png) | Miss | 0 | 0% | `400` |
+
+The hit window depends on the beatmap's [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty). A hit is then considered inside a hit window if `hit error < hit window`, meaning the value listed is half of the hit window width.
```

Comment:
if "the value listed is half of the hit window width", then the table heading row should say something like "acceptable hit error" instead since that is not a hit window